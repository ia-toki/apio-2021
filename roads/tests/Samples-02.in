c10234cabdfd6eae47773fb6f163e0350fc797e1
4
0 1 5
2 0 10
0 3 5
