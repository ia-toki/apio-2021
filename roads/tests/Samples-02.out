064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
20 10 5 0
